common/src/components/RemoteComponent.tsx

```diff
@@ -8,12 +8,12 @@ interface RemoteComponentProps {
   module: string;
 }
 
-export const RemoteComponent = ({
+export const RemoteComponent: FC<RemoteComponentProps> = ({
   url,
   scope,
   module,
   ...rest
-}: RemoteComponentProps): JSX.Element => {
+}: RemoteComponentProps) => {
```

Comment:
this can be implied, now that you've fixed the `FC<RemoteComponentProps>` syntax